src/app/shared/components/info-menu/info-menu.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { InfoMenuType } from 'shared/enum/info-menu-type';
+
+@Component({
+  selector: 'app-info-menu',
+  templateUrl: './info-menu.component.html',
+  styleUrls: ['./info-menu.component.scss']
+})
+export class InfoMenuComponent {
+  @Input() public isBigIcon: boolean;
+  @Input() public typeOfContent: InfoMenuType;
```

Comment:
Can you set a default value for the type?
We can also rename it to just `type`.